pkg/apis/forklift/v1beta1/ovirtpopulator.go

```diff
@@ -0,0 +1,33 @@
+package v1beta1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+type OvirtImageIOPopulator struct {
```

Comment:
we don't want to change the name to OvirtVolumePopulator?

Reply:
yeah, it also crossed my mind when seeing 'imageio' in other places before